addons/cluster-admin/cluster-admin.addon

```diff
@@ -1,4 +1,5 @@
 # Name: cluster-admin
```

Comment:
Also as discussed with @hferentschik @budhrg  @praveenkumar  The addon name should be "create-cluster-addon"